mmv1/products/securitycenterv2/ProjectSccBigQueryExports.yaml

```diff
@@ -0,0 +1,139 @@
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'ProjectSccBigQueryExports'
+base_url: projects/{{project}}/locations/{{location}}/bigQueryExports
+self_link: projects/{{project}}/locations/{{location}}/bigQueryExports/{{big_query_export_id}}
+create_url: projects/{{project}}/locations/{{location}}/bigQueryExports?bigQueryExportId={{big_query_export_id}}
+update_verb: :PATCH
+update_mask: true
+import_format:
+  - 'projects/{{project}}/locations/{{location}}/bigQueryExports/{{big_query_export_id}}'
+custom_code: !ruby/object:Provider::Terraform::CustomCode
+  post_import: templates/terraform/post_import/scc_v2_project_big_query_export_config.go.erb
+description: |
+  A Cloud Security Command Center (Cloud SCC) Big Query Export Config.
+  It represents exporting Security Command Center data, including assets, findings, and security marks
+  using gcloud scc bqexports
+  ~> **Note:** In order to use Cloud SCC resources, your organization must be enrolled
+  in [SCC Standard/Premium](https://cloud.google.com/security-command-center/docs/quickstart-security-command-center).
+  Without doing so, you may run into errors during resource creation.
+references: !ruby/object:Api::Resource::ReferenceLinks
+  guides:
+    'Official Documentation': 'https://cloud.google.com/security-command-center/docs/how-to-analyze-findings-in-big-query'
+  api: 'https://cloud.google.com/security-command-center/docs/reference/rest/v2/projects.locations.bigQueryExports'
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'scc_v2_project_big_query_export_config_basic'
+    primary_resource_id: 'custom_big_query_export_config'
+    external_providers: ["random", "time"]
+    skip_test: true
+    vars:
+      big_query_export_id: 'my-export'
+      dataset: 'my-dataset'
+      dataset_id: 'my_dataset_id'
+      name: 'my-export'
+    test_env_vars:
+      org_id: :ORG_ID
+      project: :PROJECT_NAME
+    ignore_read_extra:
+      - 'project'
+
+parameters:
+  - !ruby/object:Api::Type::String
+    name: bigQueryExportId
+    required: true
+    immutable: true
+    url_param_only: true
+    description: |
+      This must be unique within the organization.
+  - !ruby/object:Api::Type::String
+    name: location
+    immutable: true
+    url_param_only: true
+    default_value: global
+    description: |
+      location Id is provided by organization. If not provided, Use global as default.
+properties:
+  - !ruby/object:Api::Type::String
+    name: name
+    description: |
+      The resource name of this export, in the format
+      `projects/{{project}}/locations/{{location}}/bigQueryExports/{{big_query_export_id}}`.
+      This field is provided in responses, and is ignored when provided in create requests.
```

Comment:
Name appears like it should be output only (`output:true`).

Reply:
Addressed